Installation/changes.html

```diff
@@ -143,7 +142,15 @@ <h2 id="release4.10">Release 4.10 </h2>
 <!-- Polygons -->
 <!-- Cell Complexes and Polyhedra -->
 <!-- Arrangements -->
+
 <!-- Triangulations and Delaunay Triangulations -->
+<h3> 2D Triangulation data structure </h3>
+  <ul>
+    <li>
+      Add function <code>insert_in_hole</code>.
+    </li>
+  </ul>
+
```

Comment:
The `changes.html` no longer has conflicts with `master`, and the modifications are at the right place in the file.